src/handlers/Command.js

```diff
@@ -37,7 +38,10 @@ class CommandHandler {
 
         _this.client.on("ready", async () => {
             const rest = new REST({ version: "9" }).setToken(_this.client.token);
-
+            if (_this.options.notifyOwner && !_this.client.msgs.notifyOwnerMessage) throw new Error("[OPCommands] Missing notifyOwnerMessage parameter.");
+            if (_this.options.notifyOwner) _this.client.users.fetch(_this.client.owners[0]).then(user => {
+                user.send(_this.client.msgs.notifyOwnerMessage);
```

Comment:
Here you should do something like this:
```js
if (_this.client.msgs.notifyOwnerMessage) {
  _this.client.msgs.notifyOwnerMessage(user); // because it should be a function
} else {
  // if there isn't any message, it uses a default one
  user.send("I'm online!");
};
```